ocaml/xapi/helpers.ml

```diff
@@ -72,7 +72,10 @@ let call_script ?(log_output = Always) ?env ?stdin script args =
     | On_failure ->
         (false, true)
   in
-  debug "about to call script: %s %s" script
+  let timeout_msg =
+    if Option.is_none timeout then " without a timeout" else ""
+  in
+  debug "about to call script%s: %s %s" timeout_msg script
```

Comment:
Include the timeout duration in the log message?

Reply:
Yeah, makes sense, this way we can try looking for the failure of the script when triaging with the logs. I've added this